modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -7,13 +7,21 @@
 [id="registry-configuring-registry-storage-swift-trust_{context}"]
 = Configuring the Image Registry Operator to trust Swift storage
```

Comment:
The title of this part need update too.

Reply:
@xiuwang What do you think? "Configuring the Image Registry for Red Hat OpenStack Platform Swift Storage"

Thanks!

Reply:
How about "Manage the Image Registry Operator Redirects" ?

Reply:
@xiuwang Slightly changed, but updated. PTAL! Thanks!